ShareFile rebranding to Progress (#1887)
MicrosoftDocs/entra-docs: docs/identity/saas-apps/sharefile-tutorial.md

---
title: Configure Progress ShareFile for Single sign-on with Microsoft Entra ID
description: Learn how to configure single sign-on between Microsoft Entra ID and Progress ShareFile.
author: nguhiu
manager: mwongerapk
ms.reviewer: celested
ms.topic: how-to
ms.date: 05/20/2025
ms.custom: sfi-image-nochange
# Customer intent: As an IT administrator, I want to learn how to configure single sign-on between Microsoft Entra ID and Progress ShareFile so that I can control who has access to Progress ShareFile, enable automatic sign-in with Microsoft Entra accounts, and manage my accounts in one central location.
---
# Configure Progress ShareFile for Single sign-on with Microsoft Entra ID

> In this article,  you learn how to integrate Progress ShareFile with Microsoft Entra ID. When you integrate Progress ShareFile with Microsoft Entra ID, you can:

Reviewer: **[Pre-existing]** Double space after the comma. Should be a single space.

```suggestion
In this article, you learn how to integrate Progress ShareFile with Microsoft Entra ID. When you integrate Progress ShareFile with Microsoft Entra ID, you can:
```

> In this article,  you configure and test Microsoft Entra single sign-on in a test environment.

Reviewer: **[Pre-existing]** Double space after the comma. Should be a single space.

```suggestion
In this article, you configure and test Microsoft Entra single sign-on in a test environment.
```

> 	![Screenshot shows the Certificate download link](common/certificatebase64.png "Certificate")

Reviewer: **[Pre-existing]** Alt text is missing a trailing period. Screen readers use the period to identify the end of alt text.

```suggestion
	![Screenshot shows the Certificate download link.](common/certificatebase64.png "Certificate")
```

4. Select **Add User**.
  
	>[!NOTE]

> 	>The Microsoft Entra account holder will receive an email and follow a link to confirm their account before it becomes active.You can use any other Progress ShareFile user account creation tools or APIs provided by Progress ShareFile to provision Microsoft Entra user accounts.

Reviewer: Missing space after the period before "You can use" ("active.You" should be "active. You").

```suggestion
	>The Microsoft Entra account holder will receive an email and follow a link to confirm their account before it becomes active. You can use any other Progress ShareFile user account creation tools or APIs provided by Progress ShareFile to provision Microsoft Entra user accounts.
```

## Test SSO 

In this section, you test your Microsoft Entra single sign-on configuration with following options.

* Select **Test this application**, this option redirects to Progress ShareFile Sign-on URL where you can initiate the login flow.

* Go to Progress ShareFile Sign-on URL directly and initiate the login flow from there.

* You can use Microsoft My Apps. When you select the Progress ShareFile tile in the My Apps, this option redirects to Progress ShareFile Sign-on URL. For more information, see [Microsoft Entra My Apps](/azure/active-directory/manage-apps/end-user-experiences#azure-ad-my-apps).

## Related content

> Once you configure Progress ShareFile you can enforce session control, which protects exfiltration and infiltration of your organization’s sensitive data in real time. Session control extends from Conditional Access. [Learn how to enforce session control with Microsoft Defender for Cloud Apps](/cloud-app-security/proxy-deployment-any-app).

Reviewer: Missing comma after the introductory clause "Once you configure Progress ShareFile."

```suggestion
Once you configure Progress ShareFile, you can enforce session control, which protects exfiltration and infiltration of your organization's sensitive data in real time. Session control extends from Conditional Access. [Learn how to enforce session control with Microsoft Defender for Cloud Apps](/cloud-app-security/proxy-deployment-any-app).
```
